django_guid/integrations.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from django.core.exceptions import ImproperlyConfigured
+
+logger = logging.getLogger('django_guid')
+
+
+class SentryIntegration:
+    """
+    This integrations, when added to the django_guid settings,
+    ensures that each requests correlation ID is passed on to Sentry exception logs.
```

Comment:
Docstring probably needs to be revisited 🍡

Reply:
done